src/ceph-disk/ceph_disk/main.py

```diff
@@ -3178,6 +3224,12 @@ def activate(
         raise Error('No OSD uuid assigned.')
     LOG.debug('OSD uuid is %s', fsid)
 
+    juuid = read_one_line(path, 'journal_uuid')
```

Comment:
It's legitimate to not have a journal in some cases. If that's the case, it should not fail.